webrtc-stats.html

```diff
@@ -73,8 +73,8 @@
         <li>It MUST support generating statistics for the type RTCOutboundRTPStreamStats, with
         attributes packetsSent, bytesSent.
         </li>
-        <li>For all subclasses of RTCRtpStreamStats, it MUST include ssrc, associateStatsId,
-        isRemote, mediaType.
+        <li>For all subclasses of RTCRTPStreamStats, it MUST include ssrc, mediaType,
+        and either remoteId or localId as appropriate.
```

Comment:
Thinking about this a little more .... if the implementation doesn't support remote stats, or the first RR / SR hasn't arrived yet, it's not logical to have the remote stats, and then remoteId doesn't make sense.

Suggested rephrase: "MUST include ssrc and mediaType. When stats exist for both sides of a connection, in the form of an inbound-rtp / remote-outbound-rtp pair or an outbound-rtp / remote-inbound-rtp pair, the members "remoteId" and "localid" MUST be present".